redbot/core/core_commands.py

```diff
@@ -3393,6 +3393,54 @@ async def count_ignored(self, ctx: commands.Context):
         )
         return msg
 
+    @commands.command(
+        cls=commands.commands._IsTrueBotOwner, name="sudo",
+    )
+    async def sudo(self, ctx: commands.Context):
+        """Enable your bot owner privileges.
+
+        Sudo permission is auto removed after interval set with `[p]sudotimer` (Default to 15 minutes).
+        """
+        if ctx.author.id not in self.bot.owner_ids:
+            self.bot.owner_ids.add(ctx.author.id)
+            await ctx.send(_("Your bot owner privileges have been enabled."))
+            await asyncio.sleep(delay=await self.bot._config.sudotime())
+            self.bot.owner_ids.discard(ctx.author.id)
```

Comment:
This might make more sense as a cancellable task - currently using `[p]sudo`, `[p]unsudo` after 5 minutes, and `[p]sudo` right after will lead to "sudo time" being shorter by 5 minutes.